tasks/set_vars.yml

```diff
@@ -1,21 +1,27 @@
 ---
-- name: Ensure ansible_facts used by role
-  setup:
-    gather_subset: min
-  when: not ansible_facts.keys() | list |
-    intersect(__template_required_facts) == __template_required_facts
+- name: "Gather the packages facts"
+  ansible.builtin.package_facts:
+    manager: rpm
 
-- name: Set platform/version specific variables
-  include_vars: "{{ __vars_file }}"
-  loop:
-    - "{{ ansible_facts['os_family'] }}.yml"
-    - "{{ ansible_facts['distribution'] }}.yml"
-    - >-
-      {{ ansible_facts['distribution'] ~ '_' ~
-      ansible_facts['distribution_major_version'] }}.yml
-    - >-
-      {{ ansible_facts['distribution'] ~ '_' ~
-      ansible_facts['distribution_version'] }}.yml
-  vars:
-    __vars_file: "{{ role_path }}/vars/{{ item }}"
-  when: __vars_file is file
+- name: "Check postgresql.conf"
+  stat:
+    path: /var/lib/pgsql/data/postgresql.conf
+  register: __postgresql_conf
+
+- name: "Check TLS crt file"
+  stat:
+    path: "/etc/pki/tls/certs/{{ cert_name }}.crt"
+  register: __postgresql_cert
+
+- name: "Check TLS private key file"
+  stat:
+    path: "/etc/pki/tls/private/{{ cert_name }}.key"
+  register: __postgresql_key
+
+- name: "Supported streams in RHEL8"
```

Comment:
For variables used internally by the role that the user should not change and that have a fixed value, use `vars/main.yml`, and use this naming convention:
```
__postgresql_versions_el8: ["10", "12", "13"]
__postgresql_versions_el9: ["13"]
```
We prefer to use `el` instead of `rhel` if it is applicable to any RHEL os family - centos, alma, rocky, etc.